lib/packagekit-glib2/pk-common.c

```
@@ -361,3 +363,36 @@ pk_get_distro_version_id (GError **error)
+ * Get the test data directory (or NULL if not in test mode)
+ *
+ * Returns: (nullable): the test data directory (or NULL if not in test mode)
+ **/
+const gchar
+*pk_get_test_data_dir (void)
+{
+	return test_data_directory;
+}
+
+void
+pk_set_test_data_dir (const gchar *test_data_dir)
```

Cppcheck (2.20.0)

[style] The function 'pk_set_test_data_dir' is never used.

(unusedFunction)